### PR TITLE
Remove check_version in common_functions.sh

### PR DESCRIPTION
**Pull request by annaibm**
resolves : adoptium/aqa-tests#4949

### CODE REVIEW

**Comment by annaibm** [comment]
Grinder test:
https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/37316/

**Comment by llxia** [comment]
I think https://github.com/adoptium/aqa-tests/pull/4954/files#diff-e1cbec3d1700bf35e3daef146d4667564133f0a528b700594195c7a0184e7ff5L17 should also be removed. Please check it references.

File: external/build_image.sh [hunk 1]
```diff
@@ -42,7 +42,7 @@ check_external_custom=$9
 if [[ "${check_external_custom}" == "0" ]]; then
 	set_test $1
 fi
-set_version $2
```
Comment (karianna) [inline external/build_image.sh:45]: Was this deliberate?

File: external/common_functions.sh [hunk 2]
```diff
@@ -32,214 +32,190 @@ supported_builds="full"
...
-    if (check_version ${version}); then
-        echo "ERROR: Invalid Version: ${version}"
-        echo "Usage: $0 [${supported_versions}]"
-        exit 1
-    fi
-}
-
 function check_os() {
-    os=$1
+	os=$1
```
Comment (karianna) [inline external/common_functions.sh:36]: Seems to be lots of unrelated whitespace changes..

**Review by karianna — CHANGES_REQUESTED** [review]

File: external/build_image.sh (hunk 1 above)
Comment (annaibm) [reply external/build_image.sh:45]: yes, as we don't need the version check.S removed the set_version function.

File: external/common_functions.sh (hunk 2 above)
Comment (annaibm) [reply external/common_functions.sh:36]: Will check and update the unrelated spaces.

**Review by karianna — APPROVED** [review]

**Comment by llxia** [comment]
@annaibm did you test this PR in Grinder? `supported_versions` seems to be referenced in other places: https://github.com/search?q=repo%3Aadoptium%2Faqa-tests%20supported_versions&type=code

**Comment by annaibm** [comment]
@llxia  I have tested in Grinders : 
1.[37316](https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/37316/), 
2.[37315](https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/37315/). 
Other than the above,  Is there any other specific tests for external that is crucial to be done?

**Comment by llxia** [comment]
The suggestion was to remove `supported_versions`. However, `supported_versions` is still in the PR. Please ensure the Grinder is testing the correct change.

**Comment by annaibm** [comment]
Grinder tests:
[37729](https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/37729/) - aot_test
[37733](https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/37732/) - criu-functional

**Comment by annaibm** [comment]
Grinder test: 
dev.external -> [37738](https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/37738/console)

File: external/build_all.sh [hunk 3]
```diff
@@ -25,27 +25,24 @@ cleanup_images
...
+			for package in ${supported_packages}
+			do
+				for build in ${supported_builds}
+				do
+					echo "==============================================================================="
+					echo "                                                                               "
+					echo " Building Docker Images for ${test} ${vm} ${os} ${package} ${build} "
+					echo "                                                                               "
+					echo "==============================================================================="
+					$(dirname "$0")/build_image.sh ${test} ${vm} ${os} ${package} ${build}
```
Comment (LongyuZhang) [inline external/build_all.sh:41]: If the `version` variable is still available here, we can still print it here so we can have output about which version we are building.

**Review by LongyuZhang — APPROVED** [review]
LGTM

**Comment by LongyuZhang** [comment]
> Grinder test: dev.external -> [37738](https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/37738/console)

The failure is not related to this PR, will fix in other PRs.
